android/engine/src/main/java/org/smartregister/fhircore/engine/util/PractitionerDetailsUtil.kt

```diff
@@ -19,22 +19,31 @@ package org.smartregister.fhircore.engine.util
 import ca.uhn.fhir.context.FhirContext
 import ca.uhn.fhir.parser.IParser
 import org.smartregister.model.location.LocationHierarchy
+import org.smartregister.model.practitioner.FhirCareTeamExtension
+import org.smartregister.model.practitioner.FhirOrganizationExtension
 import org.smartregister.model.practitioner.FhirPractitionerDetails
 import org.smartregister.model.practitioner.KeycloakUserDetails
 import org.smartregister.model.practitioner.PractitionerDetails
 import org.smartregister.model.practitioner.UserBioData
 
-object FhirContextUtil {
+object PractitionerDetailsUtil {
 
-  fun getPractitionerDetailParser(): IParser {
-    val fhirContext = FhirContext.forR4Cached()
+  private const val PRACTITIONER_DETAILS_ENDPOINT = "practitioner-details"
+  private const val PRACTITIONER_DETAILS_KEYCLOAK_UUID_PARAMETER = "keycloak-uuid"
 
-    fhirContext.registerCustomType(PractitionerDetails::class.java)
-    fhirContext.registerCustomType(FhirPractitionerDetails::class.java)
-    fhirContext.registerCustomType(LocationHierarchy::class.java)
-    fhirContext.registerCustomType(KeycloakUserDetails::class.java)
-    fhirContext.registerCustomType(UserBioData::class.java)
+  fun getUrl(keycloakUuid: String) = "$PRACTITIONER_DETAILS_ENDPOINT?$PRACTITIONER_DETAILS_KEYCLOAK_UUID_PARAMETER=$keycloakUuid"
 
-    return fhirContext.newJsonParser()
+  private val fhirR4Cached: FhirContext = FhirContext.forR4Cached()
+
+  fun getCustomJsonParser(): IParser {
+    return fhirR4Cached.apply {
+      registerCustomType(FhirCareTeamExtension::class.java)
+      registerCustomType(FhirOrganizationExtension::class.java)
```

Comment:
@dubdabasoduba 
These classes are missing from the library, even though I already added the imports.

Reply:
Resolved